docs/index.md

```diff
@@ -1,11 +1,13 @@
-# Ralph NG
+# Ralph 3
+
+![](https://user-images.githubusercontent.com/552398/47593757-d8b76280-d978-11e8-87c6-d4ba98d26f2a.png)
```

Comment:
wouldn't it be better to store this image in our repo?

Reply:
Fixed